packages/wallet-sdk/src/sign/walletlink/relay/connection/WalletLinkConnection.ts

```diff
@@ -354,7 +354,7 @@ export class WalletLinkConnection {
       JSON.stringify({
         ...unencryptedData,
         origin: location.origin,
-        relaySource: window.coinbaseWalletExtension ? 'injected_sdk' : 'sdk',
```

Comment:
can we keep this? we just want to check `coinbaseWalletExtension` exists

Reply:
added back